docs/organizations/managing-security-and-risk.md

```diff
@@ -407,6 +407,9 @@ Security and risk management supports checking the languages and infrastructure-
 
 ## Supported security categories
 
+!!! note
+    Due to a recent update, some issues may be temporarily assigned the **Not yet categorized** category. To categorize these issues, you can [reanalyze the default branch of the relevant repository](../faq/repositories/how-do-i-reanalyze-my-repository.md#reanalyzing-a-branch). For a list of repositories that have issues with this category, use the **Security category** filter on the [Findings](#item-list) page.
```

Comment:
NOTE: I moved this right under the title so that the user sees it first when landing from the app.